src/macaron/slsa_analyzer/registry.py

```diff
@@ -434,7 +606,14 @@ def scan(self, target: AnalyzeContext, skipped_checks: list[SkippedInfo]) -> dic
                     return results
                 next_check: BaseCheck = check_queue.get()
 
-                # Look up check results to see if this check should be run
+                # Don't run excluded checks
+                if next_check.check_info.check_id not in self.checks_to_run:
+                    logger.debug("Check %s is disabled from user configutation.", next_check.check_info.check_id)
```

Comment:
```suggestion
                    logger.debug("Check %s is disabled by user configuration.", next_check.check_info.check_id)
```

Reply:
Addressed in https://github.com/oracle/macaron/pull/254/commits/63b2883b703ee2b96584f00cb34f560f49a87db9